src/Sarif/NotYetAutoGenerated/ReportingDescriptor.cs

```diff
@@ -41,87 +41,88 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// A stable, opaque identifier for the report.
         /// </summary>
-        [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false, Order = 1)]
```

Comment:
>1 [](http://example.com/codeflow?start=87&length=1)

how I get the current order:

I use the version before issue, and then temp change code set all fields always written even it is null, so that I can get the full list of the ordering, and then use the same ordering.

fyi it was like below:

{
              "id": "BA2001",
              "fullDescription": {
                "text": "64-bit images should"
              },
              "helpUri": "[https://gith](https://gith)",
              "help": {
                "text": "64-bit ima"
              },
              "messageStrings": {
                "Pass": {
                  "text": "'{0}' is"
                },
                "Error": {
                  "text": "'{0}' is"
                },
                "NotApplicable_InvalidMetadata": {
                  "text": "'{0}' was"
                }
              },
              "shortDescription": null,
              "name": "LoadImageAboveFourGigabyteAddress",
              "deprecatedIds": null,
              "guid": null,
              "deprecatedGuids": null,
              "deprecatedNames": null,
              "defaultConfiguration": {},
              "relationships": null,
              "properties": {
                "equivalentBinScopeRuleReadableName": "FourGbCheck"
              }
            }

Reply:
Thanks for being so thorough here with your explanation. I found this super helpful.